tool/gravity/cli/install.go

```diff
@@ -793,10 +793,14 @@ var InterruptSignals = signals.WithSignals(
 func NewInstallerConnectStrategy(env *localenv.LocalEnvironment, config InstallConfig, commandArgs cli.CommandArgs) (strategy installerclient.ConnectStrategy, err error) {
 	commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
 		cli.NewFlag("token", config.Token),
-		cli.NewBoolFlag("selinux", config.SELinux),
 		cli.NewBoolFlag("from-service", true),
 		cli.NewArg("path", config.StateDir),
 	)
+	if config.Mode != constants.InstallModeInteractive {
```

Comment:
I'm probably missing something, but doesn't this basically silently discard selinux configuration if trying to use the interactive installer?

Reply:
It is not discarding the SELinux configuration - the interactive installer is special in a way that it only installs "remotely" - it, on its own, is not supposed to change the system it is running on hence it omits this configuration.

Now, you're not restricted from running the agent on the same node where wizard is running and the agent could potentially configure SELinux if necessary (note, that due to lack of UI configuration and the off default for the SELinux flag, it will not be able to do that in UI-based case unless the agent's command is manually updated to include the SELinux flag).